AudibleApi.Common/Item.cs

```diff
@@ -14,6 +14,7 @@ public bool IsEpisodes
 			=> Relationships?.Any(r => (r.RelationshipToProduct == RelationshipToProduct.Child || r.RelationshipToProduct == RelationshipToProduct.Parent) && r.RelationshipType == RelationshipType.Episode)
 			?? false;
 		public string PictureId => ProductImages?.PictureId;
+		public string PictureId_1215 => ProductImages?.PictureId_1215;
```

Comment:
Does 1215 have a different ID? The other sizes all share the same ID which is why Libation's PictureStorage can do things like `=> Path.Combine(ImagesDirectory, $"{def.PictureId}{def.Size}.jpg");`

Reply:
Yes, they do.  It seems like most titles have 2 native image sizes: 1 is 500 pixels and the other around 2000 pixels.  Depending on which size you query with SL###, Audible will serve you an image REDUCED to that size. But it will not serve you an image larger than the the Picture ID's native size.

Look at these returned values as an example:

        "1215": "https://m.media-amazon.com/images/I/91Atrdr7spL._SL1215_.jpg",
        "252": "https://m.media-amazon.com/images/I/51nULUFT+qL._SL252_.jpg",
        "315": "https://m.media-amazon.com/images/I/51nULUFT+qL._SL315_.jpg",
        "360": "https://m.media-amazon.com/images/I/51nULUFT+qL._SL360_.jpg",
        "408": "https://m.media-amazon.com/images/I/51nULUFT+qL._SL408_.jpg",
        "500": "https://m.media-amazon.com/images/I/51nULUFT+qL._SL500_.jpg",
        "558": "https://m.media-amazon.com/images/I/91Atrdr7spL._SL558_.jpg",
        "570": "https://m.media-amazon.com/images/I/91Atrdr7spL._SL570_.jpg",
        "882": "https://m.media-amazon.com/images/I/91Atrdr7spL._SL882_.jpg",
        "900": "https://m.media-amazon.com/images/I/91Atrdr7spL._SL900_.jpg"

There are only 2 picture IDs. If you don't specify a size, you can get the native image size:

https://m.media-amazon.com/images/I/51nULUFT+qL.jpg
https://m.media-amazon.com/images/I/91Atrdr7spL.jpg

The first one is native 500x500 and the second one is native 2400x2400